_authors/osci_2023/ikibo.markdown

```diff
@@ -0,0 +1,9 @@
+---
+name: Oleg Kravchuk
+short_name: ikibo
+github: ikibo
+linkein: olegkravchuk
```

Comment:
Hey this tag should read `linkedin`, please update and then we will be good!

Reply:
thnx @dtaivpp , fixed